tsconfig.json

```diff
@@ -1,10 +1,14 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["ES6"],
+    "lib": ["dom", "ES6"],
```

Comment:
I actually don't think this is needed

Reply:
ooh yeah I think you're right with the addition of the `skipLibCheck`!